docs-src/tctl-next/batch/index.md

```diff
@@ -24,11 +24,11 @@ You can use this Job ID in the `tctl batch describe` command, which describes th
 You can also use the Job ID to terminate the batch job itself.
 Terminating a batch job does not roll back the operations already performed by the batch job.
 
-### tctl batch commands
+### tctl batch-v2 commands
 
-- [tctl batch describe](/tctl-next/batch#describe)
-- [tctl batch list](/tctl-next/batch#list)
-- [tctl batch terminate](/tctl-next/batch#terminate)
+- [tctl batch describe --job_id](/tctl-next/batch-v2#describe)
```

Comment:
in tctl next the flag is

tctl next also doesn't have `batch-v2` command, only `batch`

```suggestion
- [tctl batch describe --job-id](/tctl-next/batch#describe)
```